<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOctrTree_my</class>
 <widget class="QWidget" name="QOctrTree_my">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QOctrTree_my</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>317</width>
     <height>323</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSlider" name="slider">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="value">
       <double>2.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_SearchFirst">
      <property name="text">
       <string>体素搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_SearchSecond">
      <property name="text">
       <string>k领域搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_SearchThird">
      <property name="text">
       <string>半径搜索</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>160</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>层数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>邻近点数：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>半径：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>X：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Y：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Z:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>开始搜索：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>49</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>180</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>鼠标选点</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
